specs/CHANGELOG.md

```diff
@@ -1,6 +1,20 @@
 # Change Log
 
-All notable changes to this project will be documented in this file. See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+## 0.1.4 (2023-09-19)
+
+
+### Bug Fixes
+
+* **prettier:** consistent version ([#5850](https://github.com/algolia/instantsearch/issues/5850)) ([ca59c6d](https://github.com/algolia/instantsearch/commit/ca59c6dbd5c9eac4e2e0179a24e39bca997ae141))
+* **searchbox:** add aria-hidden to svg icons ([#5547](https://github.com/algolia/instantsearch/issues/5547)) ([50344e3](https://github.com/algolia/instantsearch/commit/50344e3b14c22c886415c0e7d799aca778dc39ab)), closes [#5546](https://github.com/algolia/instantsearch/issues/5546)
+* **svg:** don't style width/height in attributes with unit ([#5550](https://github.com/algolia/instantsearch/issues/5550)) ([31b85a6](https://github.com/algolia/instantsearch/commit/31b85a6ad56993455adb201f88ab1d1ae2d96683))
```

Comment:
Similar to the previous release, this should not be here, it is part of 0.1.3.
```suggestion
```

Reply:
this is because we don't tag things that don't go on npm, maybe this changelog should be ignored like the examples?

Reply:
Ah yes, we could do that from now on.